public/resources/tutorial/tutorial.css

```diff
@@ -135,6 +135,10 @@ li {
   margin: 0 0 10px 0;
 }
 
+img{
```

Comment:
Please add a space between the tag name and the open bracket like this...

``img {``